cpp/src/strings/copying/concatenate.cu

```diff
@@ -63,37 +64,30 @@ struct chars_size_transform {
 
 auto create_strings_device_views(std::vector<column_view> const& views, cudaStream_t stream)
 {
+  CUDF_FUNC_RANGE();
   // Create device views for each input view
-  using CDViewPtr =
-    decltype(column_device_view::create(std::declval<column_view>(), std::declval<cudaStream_t>()));
-  auto device_view_owners = std::vector<CDViewPtr>(views.size());
-  std::transform(
-    views.cbegin(), views.cend(), device_view_owners.begin(), [stream](auto const& col) {
-      return column_device_view::create(col, stream);
-    });
-
+  cudf::thread_range r1{"device_view_owners"};
+  cudf::thread_range r2{"create_contiguous_device_views"};
```

Comment:
Are the `cudf::thread_range` statements for nvtx tracing? Should these be removed?

Reply:
Yes. they are removed.